### PR TITLE
Graphs

### PR DESCRIPTION
**Pull request by wilblack**
Hey @jennywalsh and @sethrh please take a look at this pull request. Just kind of testing how they work. Leave a comment to let me know you've seen it. Seth the sorting we talked about it in the survey.models if you want to see how it ended up.

### CODE REVIEW

File: server/apps/survey/models.py [hunk 1]
```diff
@@ -431,6 +459,11 @@ def question_types(self):
     def report_types(self):
         return REPORT_TYPE_CHOICES
 
+    def sort_by_rows(self, x):
```
Comment (wilblack) [inline server/apps/survey/models.py:462]: Here is the mapping function I defined. It's name kind of sucks.

File: server/apps/survey/models.py [hunk 2]
```diff
@@ -476,7 +509,16 @@ def get_answer_domain(self, survey, filters=None):
             return (MultiAnswer.objects.filter(response__in=answers)
                                        .values('answer_text')
                                        .annotate(surveys=Count('answer_text')))
+        elif self.rows:
+
+            res = (answers.values('answer')
+                           .annotate(locations=Sum('respondant__locations'), surveys=Count('answer')))
+            out = sorted(res, key=self.sort_by_rows)
```
Comment (wilblack) [inline server/apps/survey/models.py:516]: And here is where the actual sorted call takes place. These line comments are kind of cool.

**Comment by jennywalsh** [comment]
Line comments are very cool and I like the review help they give the reviewer.  I will leave it up to you and Seth to coordinate the merge!

**Comment by wilblack** [comment]
Thanks Jenny. I'll merge this now and push to staging.

**Comment by jennywalsh** [comment]
Cool.  Let me know when I can head in and start testing for our delivery on
Tuesday.

On Fri, Aug 1, 2014 at 2:35 PM, Wil Black notifications@github.com wrote:

> Merged #13 https://github.com/point97/survey-ost/pull/13.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/point97/survey-ost/pull/13#event-148358487.

## 

Jenny Walsh | _POINT 97_ | An Ecotrust Company
Product Development Director
m: +1(503)705-9391 | _web http://pointnineseven.com_ | _twitter
https://twitter.com/Point_97_ | _vimeo
http://vimeo.com/pointnineseven/videos_

**Comment by wilblack** [comment]
Will do, I ran into some inconsistencies during deployment. Cleaning those up now. Should have it ready to go within the hour. 
I'm liking the communication through the pull requests, the only problem is that github is sending these to me personal email. I'll have to figure out how to fix that.

**Comment by sethrh** [comment]
You can add another email address in your account settings, and then associate emails with organizations - https://github.com/settings/notifications

**Comment by wilblack** [comment]
Awesome, I love when a company thinks of a service before i know i need it.

On Fri, Aug 1, 2014 at 4:04 PM, Seth Hill notifications@github.com wrote:

> You can add another email address in your account settings, and then
> associate emails with organizations -
> https://github.com/settings/notifications
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/point97/survey-ost/pull/13#issuecomment-50944195.

## 

Wil Black
wilblack21@gmail.com
plus.google.com/+WilBlack

**Pull request merged**